libcontainer/message_linux.go

```diff
@@ -56,5 +56,5 @@ func (msg *Bytemsg) Serialize() []byte {
 }
 
 func (msg *Bytemsg) Len() int {
-	return syscall.NLA_HDRLEN + len(msg.Value) + 1 // null-terminated
+	return NLA_HDRLEN() + len(msg.Value) + 1 // null-terminated
```

Comment:
You mean, just hard-code it everywhere (and not have the other two new files)? I'm okay with that, but I thought everyone would want to keep the syscall.NLA_HDRLEN where supported.